src/pages/how-to/routing-traffic-to-private-networks.mdx

```diff
@@ -49,15 +51,49 @@ Nodes connected to routing peers will choose one of them to route packets to ext
 Masquerade hides other NetBird network IPs behind the routing peer local address when accessing the target Network range. This option allows access to your private networks without configuring routes on your local routers or other devices.
 
 If you don't enable this option, you must configure a route to your NetBird network in your external network infrastructure.
+
+### DNS Routes
+An alternative to specifying a network range directly is to use DNS routes. Instead of adding the network directly, you can add multiple domains in a route that will be dynamically resolved on the client. The resolved IP addresses for these domains will be added as routes.
+
+The interval for resolving DNS routes is 60 seconds by default, but you can change it using the `--dns-router-interval` flag:
+
+
+```bash
+netbird up --dns-router-interval 30s
+```
+
+There's also a keep routes switch that is enabled by default.
+
+<p>
+    <img src="/docs-static/img/how-to-guides/netbird-network-routes-dns-routes.png" alt="high-level-dia" className="imagewrapper-big"/>
+</p>
+
+When this switch is on and a domain no longer resolves to an IP address, the corresponding route will still be kept (and any new resolved IP addresses will be added).
+If the switch is off, then the routes will be replaced with the newly resolved IP addresses instead.
```

Comment:
If the switch is off, the routes will be replaced with the newly resolved IP addresses.